knative-operator/deploy/resources/knativekafka/kafkasource-v0.17.1.yaml

```diff
@@ -28,7 +21,7 @@ rules:
   verbs: &everything
   - get
   - list
-  - watch
+  - watchals
```

Comment:
@aliok `watchals` ?

Reply:
Donno what that is...

But, compared with our eventing-contrib release file and that's the only difference in both channel and source manifest